### PR TITLE
feat(appeals): importing appellant cases (ASI-21)

### PR DESCRIPTION
**Pull request by fabfab-kc**
Imports appellant cases from the Service Bus topic.
- Validates request against schema
- Processes data and imports
- Additional bugfixes

## Issue ticket number and link

[ASI-21](https://pins-ds.atlassian.net/browse/ASI-21)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/server/endpoints/integrations/integrations.broadcasters/appeal.js [hunk 1]
```diff
@@ -7,8 +8,58 @@ export const broadcastAppeal = async (
 	/** @type {string} */ updateType = EventType.Update
 ) => {
 	if (!config.serviceBusEnabled) {
-		return false;
+		//return false;
 	}
 
 	pino.info({ appealId, updateType });
+	// const appeal = await databaseConnector.appeal.findUnique({
```
Comment (kandcdev) [inline appeals/api/src/server/endpoints/integrations/integrations.broadcasters/appeal.js:15]: leftover?

Comment (fabfab-kc) [reply appeals/api/src/server/endpoints/integrations/integrations.broadcasters/appeal.js:15]: No I'm going to fast follow with https://pins-ds.atlassian.net/browse/ASI-17, that's the code that loads the appeal to be broadcast

**Review by glpins — APPROVED** [review]

File: appeals/functions/document-processing/malware-detection/index.js [hunk 2]
```diff
@@ -34,9 +33,9 @@ export default async function (context, msg) {
 const statusFromScanResult = (scanResult) => {
 	switch (scanResult) {
 		case 'Malicious':
-			return AVSCAN_STATUS.AFFECTED;
+			return 'affeced';
```
Comment (james-hill-kc) [inline appeals/functions/document-processing/malware-detection/index.js:36]: This looks like a typo - should be `affected`?

**Review by james-hill-kc — COMMENTED** [review]
Looks good, just spotted one possible typo

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**